pallets/connectors/src/lib.rs

```diff
@@ -378,48 +400,44 @@ pub mod pallet {
 		#[pallet::call_index(5)]
 		pub fn update_member(
 			origin: OriginFor<T>,
-			domain_address: DomainAddress,
 			pool_id: PoolIdOf<T>,
 			tranche_id: TrancheIdOf<T>,
+			domain: DomainAddress,
```

Comment:
I don't really agree with this renaming because it's not a `Domain`, it's a `DomainAddress`. At worse, we could name it `address`, not `domain`.